apps/accounts/filters.py

```diff
@@ -104,11 +105,11 @@ def filter_latest(queryset, name, value):
 
         if name == "latest_secret_change_failed":
             queryset = queryset.filter(date_change_secret__gt=date).exclude(
-                change_secret_status="ok"
+                change_secret_status=ChangeSecretRecordStatusChoice.success
             )
 
         if kwargs:
-            queryset = queryset.filter(date_last_login__gte=date)
+            queryset = queryset.filter(**kwargs)
         return queryset
 
     @staticmethod
```

Comment:
I'm sorry, but it seems there's no code provided to analyze as per your request. Please share the code you want to check for potential issues or improvements.